include/dlaf/eigensolver/tridiag_solver/kernels.cu.h

```diff
@@ -0,0 +1,60 @@
+//
```

Comment:
Should this file not be just `kernels.h` (or `gpu_kernels.h`, or `gpu/kernels.h`)? It's not using any device functionality as far as I can tell. I would reserve the `*.cu*` for files that actually use device constructs.

Reply:
I just wanted to be nice to reviewer avoiding moving kernel.cu.... 😛